.github/workflows/ci.yml

```diff
@@ -2,25 +2,8 @@ name: CI
 
 on: [push, pull_request]
 
-jobs:
-  skip_concurrent_build:
-    name: Skip Concurrent Builds
-    runs-on: ubuntu-latest
-    outputs:
-      should_skip: ${{ steps.skip_check.outputs.should_skip }}
-    steps:
-      - uses: fkirc/skip-duplicate-actions@v5.3.0
-        id: skip_check
-        with:
-          concurrent_skipping: 'same_content'
-          github_token: ${{ github.token }}
-          do_not_skip: '["pull_request"]'
-          cancel_others: 'true'
-          skip_after_successful_duplicate: 'false'
-          
+jobs:  
   build:
-    needs: skip_concurrent_build
-    if: ${{ needs.skip_concurrent_build.outputs.should_skip != 'true' }}
```

Comment:
@s4na would you mind bringing this back? I'm totally fine merging this afterwards

Reply:
OK